src/main/kotlin/frc/robot/subsystems/VisionToggleSubsystem.kt

```diff
@@ -0,0 +1,13 @@
+package frc.robot.subsystems
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+
+enum class VisionModes { BALL, HIGHGOAL }
+
+class VisionToggleSubsystem() : SubsystemBase() {
+    var visionMode = VisionModes.BALL
```

Comment:
I'm not sure it makes sense to make the vision mode (ball/high goal) a toggle. The driver has to remember which mode the robot is in, may have to switch frequently, and could get bad results if they choose the wrong mode.

It probably makes more sense to just assign high goal vision and ball vision to separate buttons. You could move climbing to the D-Pad, use the triggers for vision (ie - right high goal, left ball), and keep shooting on the right bumper (or something like that)

Reply:
Ok. I've sent you my thoughts on discord and may update the controls again before Thursday